core/src/test/scala/org/apache/spark/sql/store/CreateIndexTest.scala

```diff
@@ -49,13 +50,16 @@ class CreateIndexTest extends SnappyFunSuite with BeforeAndAfterEach {
   override def afterAll(): Unit = {
     System.clearProperty("org.codehaus.janino.source_debugging.enable")
     System.clearProperty("spark.testing")
+    System.clearProperty(io.snappydata.Property.EnableExperimentalFeatures.configEntry.key)
     super.afterAll()
   }
 
   override def afterEach(): Unit = {
     try {
       val snContext = context.getAndSet(null)
       if (snContext != null) {
+        snContext.setConf(io.snappydata.Property.EnableExperimentalFeatures.configEntry.key,
+          io.snappydata.Property.EnableExperimentalFeatures.configEntry.defaultValueString)
```

Comment:
Just this should work, no? `snContext.setConf(io.snappydata.Property.EnableExperimentalFeatures.name, "false")`
I don't think explicit passing of defaultValueString is required.